.gitignore

```
@@ -56,3 +56,5 @@ _trial_temp*/
 ruby/.idea
 ruby/Gemfile.lock
 ruby/test/test.txt
+
+.env
```

I prefer this, since we have different varients of env files
```suggestion
.env*
```